lib/node_modules/@stdlib/blas/ext/base/dsumkbn2/lib/index.js

```diff
@@ -39,7 +39,7 @@
 * var dsumkbn2 = require( '@stdlib/blas/ext/base/dsumkbn2' );
 *
 * var x = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
-* var N = floor( x.length / 2 );
+* var N = 3;;
```

Comment:
@Planeshifter `floor` should be have been removed. Would be good to ensure that ESLint can run on these examples.